datasets/gov/vstt/saugomu_ter_kadastras.csv

```diff
@@ -0,0 +1,716 @@
+,,,,,saltiniai,string,,,,2,open,,Duomenų  šaltiniai,Nurodomi moksliniai tyrimai ir publikacijos
+,,,,,tarpt_svarba_status,integer,,,,2,open,,Teritorijos statusas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statusas
+,,,,,tarpt_svarba_apras,string,,,,3,open,,Aprašas,Saugomai teritorijai arba jos daliai suteiktas tarptautinės svarbos  (įskaitant Europos Bendrijų svarbos) teritorijos statuso aprašymas
+,,,,,duomen_monitor,string,,,,2,open,,Monitoringas,Aplinkos stebėsenos (monitoringo) duomenys
+,,,,,buf_zona_apras,string,,,,2,open,,Buferinė zona,Duomenys apie valstybinio parko buferinę apsaugos zoną
+,,,,,pastabos,string,,,,2,open,,Pastabos,
+,,,,,id_vart,integer,,,,2,open,,Duomenų teikėjas, Institucija/-jos atsakinga/-os už priežiūrą ir monitoringą. Duomenų teikėjų klasifikatorius
+,,,,,reiksme,integer,,,,2,open,,Objekto reikšmė,Gamtos paveldo objekto reikšmė: valstybės saugomi savivaldybės saugomi
+,,,,,steigejas_galiojantis,string,,,,2,open,,Steigėjas,Institucija kurios potvarkiu šiuo metu galioja saugoma teritorija arba jos riba
+,,,,,galiojantis,date,D,,,4,open,,Steigimo data,Šiuo metu galiojančio saugomos teritorijos steigimo teisės akto data
+,,,,,steig_ta_galiojantis,string,,,,4,open,,Teisinis statusas,Teisės aktas pagal kurį šiuo metu galioja saugomos teritorijos riba reglamentai arba šis teisės aktas yra šiuo metu galiojantis saugomos teritorijos teisinis pagrindas
+,,,,,tvarkymo_zona,string,,,,4,open,,Zona,Duomenys apie kraštovaizdžio tvarkymo zonas
+,,,,,funkc_prior_zon,string,,,,4,open,,Prioriteto zona,Duomenys apie biosferos rezervato funkcinio prioriteto zonas
+,,,,,planavimo_dok,string,,,,2,open,,Planavimo dokumentas,Duomenys apie parengtus planavimo dokumentus
+,,,,,vieta,string,,,,2,open,,Vieta,Vieta aprašomoji informacija (savivaldybė seniūnija jei miškas – papildomai miškų urėdija girininkija miško kvartalas ir sklypas)
+,,,,,geometrija,geometry,,,,4,open,,Ploto geometrija,
+,,,,,,,,,,,,,,
+,,,,BotaninisGpoPlotas,,,id,,,4,,,Botaniniai GPO Plotai,
+,,,,,id,string,,,,4,open,,Identifikavimo kodas,
+,,,,,pavadinimas,string,,,,4,open,,Gamtos paveldo objekto pavadinimas,
+,,,,,gkodas,string,,,,2,open,,InGIS geokodas,Saugomos teritorijos tipas
+,,,,,reiksme,integer,,,,2,open,,Objekto reikšmė,Gamtos paveldo objekto reikšmė: valstybės saugomi savivaldybės saugomi
+,,,,,porus,integer,,,,2,open,,Botaninių gamtos paveldo objektų porūšiai,
+,,,,,objektu_rusys,integer,,,,2,open,,Botaninių gamtos objektų rūšys,Botaninių gamtos paveldo objektų medžių ir krūmų porūšio objektų rūšys
+,,,,,kult_paveld_kodas,integer,,,,2,open,,Kodas,Jeigu gamtos paveldo objektas kartu turi ir kultūros paveldo objekto statusą nurodomas šio objekto kodas iš Nekilnojamųjų kultūros vertybių registro
+,,,,,st_id,string,,,,2,open,,Identifikatorius,Jei gamtos paveldo objektas yra saugomoje teritorijoje pildomas saugomos teritorijos kurioje yra gamtos paveldo objektas identifikatorius
```

Comment:
Ar identifikatorius yra neunikalus ar nepatikimas? Ar jis ateina iš kt. registro ar IS?